pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -900,6 +900,9 @@ func (r *Builder) BeforeTransferHook(c planbase.Client, vmRef ref.Ref) (ready bo
 		if snapshot.Status != "available" {
 			r.Log.Info("Snapshot not ready yet, recheking...", "snapshot", snap.Name)
 			return false, nil
+		} else if snapshot.Status == "error" {
```

Comment:
we'll never get here, no? (due to the check of `snapshot.Status != "available"`)